Signal/src/ViewControllers/ConversationView/Cells/OWSMessageFooterView.m

```diff
@@ -228,29 +228,6 @@ - (CGSize)measureWithConversationViewItem:(ConversationViewItem *)viewItem
     // Measure the actual current width, to be safe.
     CGFloat timestampLabelWidth = [self.timestampLabel sizeThatFits:CGSizeZero].width;
 
-    // Measuring the timestamp label's width is non-trivial since its
-    // contents can be relative the current time.  We avoid having
-    // message bubbles' "visually vibrate" as their timestamp labels
-    // vary in width.  So we try to leave enough space for all possible
-    // contents of this label _for the first hour of its lifetime_, when
-    // the timestamp is particularly volatile.
-    if ([DateUtil isTimestampFromLastHour:viewItem.interaction.timestamp]) {
```

Comment:
To make sure I understand - is this change intentionally re-introducing "width vibration" because it seems less bad than having overly wide bubbles with short messages (e.g. jumbomoji)

Reply:
Right.

We'll now update the cell (footer collapse = "height vibration") every ~1 minute _or_ whenever we re-enter the conversation view/app.  If re-entry triggers the update, we reset the ~1 minute countdown.  

"width vibration" should occur with around the same cadence, since the time formatting's finest resolution is 1 minute.  Most of the time, the width and height vibrations will occur _together for all_ visible cells, rather then a series of sporadic and uncoordinated animations in different cells.

Ideally, we'd also do this kind of update whenever we send/receive a message, but these updates (e.g. calling `resetContentAndLayout`) are expensive (and cost grows with, and I haven't yet found an acceptable way to do that.  This would align us with Android behavior and be a better experience.